grantpicks/serializers.py

```
@@ -0,0 +1,346 @@
+from rest_framework import serializers
+from rest_framework.serializers import ModelSerializer
+
+from accounts.serializers import SIMPLE_ACCOUNT_EXAMPLE, AccountSerializer, ProjectInPairSerializer
+from base.serializers import TwoDecimalPlacesField
+
+from .models import Project, ProjectContact, ProjectContract, ProjectRepository, Round, Vote, VotePair
```

Ruff (0.12.2)

`.models.Vote` imported but unused

Remove unused import: `.models.Vote`

(F401)